allennlp/models/constituency_parser.py

```diff
@@ -220,14 +220,14 @@ def resolve_overlap_conflicts_greedily(chosen_spans: List[Dict[str, float]]) ->
 
         Returns
         -------
-        ``chosen_spans``, with the conflicts resolved by considering
-        local differences between pairs of spans.
+        A modified list of ``spans``, with the conflicts resolved by considering local
```

Comment:
This still isn't obvious enough that you're modifying the input value.  This sentence could also describe a function that copies the input list.  Knowing that a function you're calling has side effects on its input arguments is important.